Packaging/Packaging.proj

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="CreateStepPackage" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    
+    <!-- Change this when the runner goes up in version-->
+    <BootstrapperDirName>SonarQube.MSBuild.Runner-0.9</BootstrapperDirName>
+    
+    <!-- Needed to understand the output directory of referenced projects-->
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+
+    <TaskOutputDir>$(MSBuildThisFileFullPath)\..\..\VsoBuildSteps\$(Configuration)\</TaskOutputDir>
```

Comment:
$(MSBuildThisFileFullPath)\..\..\ is repeated multiple times. It would be better to define another suitably-named property with this value and assign it once.